### ISSUE TITLE
[Bug]: createNotification returns HTTP-Code: 400 Message: Bad Request Body: {"errors":[{}]}

### ISSUE THREAD

**Issue by ArpadGBondor** [OneSignal/onesignal-node-api#77]
### What happened?

I'm trying to implement a function that we can use to send notifications using OneSignal.
The error I'm getting makes no sense to me.

### Steps to reproduce?

```Markdown
static async createNotification(title: string, message: string, userEmail: string, languageCode: string = 'en') {
    const notification = new OneSignal.Notification();

    notification.name = 'Admin notification';
    notification.contents = {
      [languageCode]: message,
    };
    notification.headings = {
      [languageCode]: title,
    };
    notification.filters = [
      {
        field: 'tag',
        key: 'email', // we agreed to use email for the tags
        relation: '=',
        value: userEmail,
      },
    ];

    try {
      return await client.createNotification(notification);
    } catch (error: any) {
      console.error(error.message);
    }

    return 'ERROR';
  }
```

### What did you expect to happen?

Either create the notification, or give me an error message that makes sense.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ArpadGBondor** [comment]
I figured it out.

So, the documentation says to configure your app like this:

```
const configuration = OneSignal.createConfiguration({
    userKey: '<YOUR_USER_KEY_TOKEN>',
    appKey: '<YOUR_APP_KEY_TOKEN>',
});

const client = new OneSignal.DefaultApi(configuration);
```

The problem is, that the API wants the API key. Not the USER_KEY_TOKEN and not the APP_KEY_TOKEN.

once I changed the configuration, it started working:

```
const configuration = OneSignal.createConfiguration({
    appKey: '<API_KEY_TOKEN>',
});
```

**Comment by selected-pixel-jameson** [comment]
Their documentation is absolutely garbage and has been since their existence.  Who is going to setup a legit push notification service that doesn't implement a back-end solution for sending out notifications. WOW. This is unbelievable. 

Thank god you've provided this. I can't believe this has been open and unaddressed for 2 months!

**Comment by Elyytscha** [comment]
I mean it was meant as a bad joke, obviously I stumbled upon this Issue because I had the same problem.

**Comment by iAmWillShepherd** [comment]
@ArpadGBondor, I'm glad you figured this out. 

Looks like you got caught by a typo in our docs!

**What exists**

```js
const configuration = OneSignal.createConfiguration({
    userKey: '<YOUR_USER_KEY_TOKEN>',
    appKey: '<YOUR_APP_KEY_TOKEN>',
});
```

**What it should be**
```js
const configuration = OneSignal.createConfiguration({
    userKey: '<YOUR_USER_KEY_TOKEN>',
    appKey: '<YOUR_API_KEY_TOKEN>',
});
```

This is confusing indeed, and I think the best course of action is to change the name of that key from `appKey` to `apiKey`. IMO, `appKey` makes no sense because we don't have a concept for that anymore. I'll ensure I bring this up to the SDK team, as I've been caught by this before too. 

I've gone ahead and updated the docs 🙏🏽 

FYI there's a [sample project for Node](https://github.com/OneSignalDevelopers/onesignal-node-sample) that shows how to correctly set this stuff up and even use some of the functions, albeit to a limited amount.

**Comment by EskelCz** [comment]
In my case it was missing ENV keys. The error message was fixed when I downgraded to node-onesignal@1.0.0-beta7
